corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -154,19 +163,36 @@ class CaseHelper:
 
     @classmethod
     def run(cls, run_config):
-        for chunk in cls.get_sql_chunks(run_config):
-            yield from cls._yield_missing_in_es(chunk)
+        domains_with_dedicated_index = list(settings.CASE_SEARCH_SUB_INDICES.keys())
```

Comment:
I think you mentioned looking at doing this one domain at a time, is that right?  That does sound like it'd end up a lot simpler, though I haven't thought it all through.  We're already pulling in chunks of 1000 at a time, so adding a breakpoint in that for each domain doesn't sound like it'd turn this into something that struggles to run at scale.

Reply:
The chunks returned by `get_sql_chunks` are not sorted by domain. So each chunk of 1000 could have any number of different domains. Adding that order by seemed unnecessary if there is at most one domain with a dedicated domain and in the case or swiss, india or self hosted instances none.